src/Cli/dotnet/commands/dotnet-workload/search/WorkloadSearchVersionsCommand.cs

```diff
@@ -102,7 +123,7 @@ public override int Execute()
             }
             else
             {
-                var workloadSet = _installer.GetWorkloadSetContents(_workloadVersion);
+                var workloadSet = _installer.GetWorkloadSetContents(_workloadVersion.Last());
```

Comment:
Probably it should be an error if you specify two workload set versions, instead of just taking the last one.

Reply:
It is! (Via the command parser rather than the command). Last, First, and Single should all be equivalent here. I only chose Last because I'd initially been thinking about not making it an error and decided on taking the last one, then decided that wasn't the best design and ended up at the same place as you.

Reply:
I changed it to Single for clarity